test/components/redButton.spec.js

```diff
@@ -0,0 +1,24 @@
+import { RedButton } from '../../src/app/components/RedButton';
+
+describe('RedButton', () => {
```

Comment:
Brakuje najwazniejszego działania czyli kliknięcia :) 
Najsensowniejszy test były:
1. Wyrenderowac komponent.
2. Kliknąć w niego.
3. Sprawdzić czy funkcja się wywołała.

To powinno pomóc:
https://testing-library.com/docs/dom-testing-library/api-events

Reply:
Próbowałam zainstalować tę biblioteczkę, ale pojawiły się jakieś problemy. Prawdopodobnie gryzła się z już istniejącą. Spędziłam kilka godzin w Internecie, żeby znaleźć jakieś rozwiązanie, ale bez rezultatu. Czy mógłbyś może pokazać nam jak to prawidłowo zainstalować i zrobić jakiś przykład?

Reply:
W jaki sposób? Patrząc po naszym package.json to ona jest juz po prostu zainstalowana :) 
```
   "@testing-library/dom": "^7.26.6",
    "@testing-library/jest-dom": "^5.11.6",
    "@testing-library/user-event": "^12.2.2",
  ```
  
  Wiec nie zostaje nic, tylko pisać. Na takie generyczne problemy (nie specyficzne dla 1 projektu) to możesz pytać na naszym kanale albo nawet na #pytania-programowanie co Ci wyskakuje itp.